spec/game_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'game'
+require 'fake_io'
+class FakeLogic
```

Comment:
I think the naming here is confusing. FakeLogic suggests that there's a Logic class but reality is that it's the TicTacToe class. Likewise with FakeIO, it's actaully the Main_IO class.